XMonad/Util/ExtensibleState.hs

```diff
@@ -24,14 +29,18 @@ module XMonad.Util.ExtensibleState (
                               , gets
                               , modified
                               , modifiedM
+
+#ifdef TESTING
```

Comment:
Does this really want to be conditional? Looks to me like you need it any time you're upgrading from an xmonad without this change.